Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/EmojiPickerViewController.swift

```diff
@@ -81,7 +131,8 @@ extension EmojiPickerViewController: UITableViewDelegate, UITableViewDataSource
         guard let cell = tableView.dequeueReusableCell(withIdentifier: "PickerTableViewCell") as? PickerTableViewCell else {
             return UITableViewCell()
         }
-        cell.configure(with: packages[indexPath.row], downloadedPackage: downloadedPackage)
+        cell.btnDownload.addTarget(self, action: #selector(btnDownloadAction(_:)), for: .touchUpInside)
```

Comment:
@mdhanif-simformsolutions  Pls don't use outlets directly in cellForRow. Use closure or protocol instead.

Reply:
Updated this thanks.